### ISSUE TITLE
can i do bbbb (baldi's basics birthday bash) mod menu 2.0.2

### ISSUE THREAD

**Issue by Baldi89989** [Fasguy/BaldisBasicsModMenu-archive#76]
Hello fasguy, it's me again (baldi89989). Sorry to bother you but can I use your code to put it on baldi's basics birthday bash so that either are bbbb mod menu 2.0.2 because the mod menu is out of date for years. I'm good enough in c# to put the mod menu on dnspy. So can I afford to use it or not? If you don't want to, it's okay, in that case I'll delete the mod and I'll delete the videos I've made. (I guess i got the wrong page for asking).

**Comment by Fasguy** [comment]
Answer me one thing first: *Why* did you feel the need to lie and say i gave you permission, when asked by someone else? Did you think they'd just take your word for it and you're off the hook?

I've gone through the song-and-dance of "It's easier to ask for forgiveness, than to ask for permission" once before already, when oldsport ported my ModMenu behind my back. I was sort-of okay with that situation, because (at least as far i know) oldsport didn't try and lie about me giving him permission. *You* on the other hand, just straight up said i gave you permission.

**Comment by Baldi89989** [comment]
So to be honest with you, I lied about it because I thought the person was going to tell you that I used your mod menu without your permission. So that's why I said yes when he asked. But when he said "proof?" I was like "oh man, how can I?" until he came to see you and I didn't expect you to comment. I stressed and panicked and regretted saying yes. I should have confessed that I hadn't given your permission but here are the consequences. I won't do it again. I don't know if you'll forgive me or not.
